web/src/scheduled_messages_overlay_ui.js

```diff
@@ -53,7 +60,11 @@ export function handle_keyboard_events(e, event_key) {
 
 function format(scheduled_messages) {
     const formatted_msgs = [];
-    for (const msg of scheduled_messages) {
+    const sortedMessages = Object.entries(scheduled_messages).sort(
+        ([, msg1], [, msg2]) =>
+            msg1.scheduled_delivery_timestamp - msg2.scheduled_delivery_timestamp,
+    );
+    for (const [, msg] of sortedMessages) {
```

Comment:
We don't use CamelCase usually. So, let use `sorted_messages` as variable name. Wouldn't it be much clear to use Object.values here?

Reply:
Yes Object.values is a better way, I have incorporated these changes. PTAL